devtools_m2/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "drip-magento2-tests",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
```

Comment:
Are these dependencies are are they "dev dependencies".

https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c

Reply:
I was thinking that the entire application exists to run tests. So there are no "dev" dependencies, really. But I'm not sure whether there is some other difference.